tests/Doctrine/Tests/DBAL/Query/QueryBuilderTest.php

```diff
@@ -327,6 +327,9 @@ public function testEmptySelect() : void
 
         self::assertSame($qb, $qb2);
         self::assertEquals(QueryBuilder::SELECT, $qb->getType());
+
+        self::expectException(QueryException::class);
```

Comment:
This should be reflected in upgrade notes.

Reply:
... and documented! Done in a02701c601f7eaf4db99192198ef91670b4095d0.
Should I squash?

Reply:
Yes.

Reply:
Please [use](https://scrutinizer-ci.com/g/doctrine/dbal/inspections/4ab891ff-68b0-4ff6-a2cd-d5d7833a9c7d/issues/files/tests/Doctrine/Tests/DBAL/Query/QueryBuilderTest.php?status=new&orderField=path&order=asc&honorSelectedPaths=0) `$this->expectException()` instead.